src/config.ts

```diff
@@ -49,6 +50,21 @@ export default class Config {
     return MessengerType.Unknown;
   }
 
+  public static DefaultPurchaseClientType(env: Environment): PurchaseClientType {
+    switch (env) {
+      case ENVIRONMENTS.SANDBOX_DEV:
+        return PurchaseClientType.Dev;
+      case ENVIRONMENTS.ADMIN: // Currently unable to hook into the twitch pubsub system from admin
+      case ENVIRONMENTS.SANDBOX_ADMIN:
+      case ENVIRONMENTS.SANDBOX_TWITCH:
+      case ENVIRONMENTS.PRODUCTION:
+        return PurchaseClientType.Twitch;
+      case ENVIRONMENTS.SERVER:
+        return PurchaseClientType.Server;
+    }
+    return PurchaseClientType.Unknown;
+  }
```

Comment:
The `ENVIRONMENTS` things is deprecated (you can read about it in the corresponding file), but I believe there is one for when running in the unit test environment. Since you have a testing-specific type and class, you should see about adding it here.